wagn/lib/wagn/railtie.rb

```diff
@@ -0,0 +1,12 @@
+
+require 'decko/engine'
```

Comment:
Did you look at the test script on generator?  That's what I'm doing, but I'm having trouble running some of the steps I need to set-up and seed the database.